### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by vinayaknayar**
**-/-/-/-/-/-/-/-/-/-/-/-/-/-/-/-/-/   (DELETE BEFORE SUBMITTING)   -/-/-/-/-/-/-/-/-/-/-/-/-/-/-/-/-/-//-/-/-/-/-/-/-/-**

# Pull Request template 
_Please, go through these steps before you submit a PR_

- Make sure that your PR is not a duplicate.
- You have done your changes in a separate branch. Branches MUST have descriptive names that start with either the fix/ or feature/ prefixes. Good examples are: fix/signin-issue or feature/issue-templates.
- You have a descriptive commit message with a short title (first line).
- Your pull request MUST NOT target the master branch on this repository. You probably want to target staging instead.
- Give a descriptive title to your PR and refer the issue you are fixing.
- Provide screenshots if working on visual changes. 

**IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/asetalias/Logistics-and-Event-Contributions/wiki/Contributing-to-ALiAS) file for detailed contributing guidelines.**

PLEASE REMOVE THIS TEMPLATE BEFORE SUBMITTING

Thanks for your contribution :hatching_chick:

### CODE REVIEW

**Comment by vinayaknayar** [comment]
PR refactored to astro.

File: package.json [hunk 1]
```diff
@@ -13,13 +13,20 @@
...
     "astro": "^1.0.6",
     "prettier": "^2.7.1",
     "prettier-plugin-astro": "^0.5.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "@types/react": "18.0.17",
     "sass": "^1.54.5"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.0",
```
Comment (ditsuke) [inline package.json:28]: Remove emotion since it's not being used now

**Review by ditsuke — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vinayaknayar) [reply package.json]: Done. Check now

Comment (ditsuke) [reply package.json]: Looks good

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+  alignItems: 'center',
+  $nest: {
+    '&:hover': {
+      backgroundColor: '#94a3b8',
+      cursor: 'pointer',
+    }
+  }
+});
+
+const navDiv = {
```
Comment (ditsuke) [inline src/components/Navbar.tsx:21]: As we're using typestyle, let's use it here too for consistency

File: src/components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  display: "flex",
+  justifyContent: "end",
+  gap: "1vw"
+};
+
+const Navbar = () => {
+  return (
+
+    <div style={navDiv}>
+      {/* <h1 className={navButton}>Sagar Deep</h1> */}
```
Comment (ditsuke) [inline src/components/Navbar.tsx:39]: Remove commented out code

File: src/components/Styles.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { useEffect, useRef } from "react";
+import { getStyles, setStylesTarget } from "typestyle";
+
+const Styles = () => {
```
Comment (ditsuke) [inline src/components/Styles.tsx:4]: I feel we should document why we need this component in a code comment.

File: src/pages/index.astro [hunk 5]
```diff
@@ -3,6 +3,10 @@ import BaseHead from "@/components/BaseHead.astro";
 import Header from "@/components/Header.astro";
 import Footer from "@/components/Footer.astro";
 import { SITE_TITLE, SITE_DESCRIPTION } from "@/config";
+import Navbar from "@/components/Navbar";
+import StyleTry from "@/components/TypestyleTry";
```
Comment (ditsuke) [inline src/pages/index.astro:7]: Remove this import

File: src/pages/index.astro [hunk 6]
```diff
@@ -11,7 +15,12 @@ import { SITE_TITLE, SITE_DESCRIPTION } from "@/config";
     <BaseHead title={SITE_TITLE} description={SITE_DESCRIPTION} />
   </head>
   <body>
+    <Styles/>
+    <Navbar/>
+    <StyleTry/>
```
Comment (ditsuke) [inline src/pages/index.astro:20]: And this too ;)

File: src/styles/Navbar.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@100;400&display=swap');
```
Comment (ditsuke) [inline src/styles/Navbar.css:1]: Can we remove this file? If we haven't ported all CSS to CSS-in-js let's do that and remove.

File: src/styles/global.css [hunk 8]
```diff
@@ -5,9 +5,9 @@
  */
 body {
   font-family: Verdana, sans-serif;
-  margin: auto;
-  padding: 20px;
-  max-width: 65ch;
+  /* margin: auto; */
```
Comment (ditsuke) [inline src/styles/global.css:8]: Remove the commented code.

**Review by ditsuke — CHANGES_REQUESTED** [review]

**Comment by vinayaknayar** [comment]
review now

**Comment by ditsuke** [comment]
Since a newer PR #251 is open, I'm closing this one.

**Pull request closed without merging**